README.md

```diff
@@ -1,69 +1,43 @@
+This pipeline checks that a given `*.json` file complies with an associated `*.schema.json` file. Right now it is only being used in the `build-ci` repo.
 
 ## Usage
-Using Github CLI:
-
-```
-gh workflow run build-and-push-image-base-spack.yml
-gh workflow run build-and-push-image-base-spack.yml -f spack-packages-version=v1.0.1
-gh workflow run build-and-push-image-build.yml
-```
-
-Note the base Spack image must be built before running `build-and-push-image-build.yml`.
-
-# Reusable workflows
-
-## `build-package.yml`
-Build the specified Spack package given a Docker build image.
-
-Inputs:
-* `container-registry`: The container registry base URL (e.g.: `ghcr.io`)
-* `container-name`: The container tag (e.g. `access-nri/example-image`)
-* `package-name`: The name of the spack package to be built (e.g. `access.nri.oasis3-mct`)
-* `compiler-name`: The name of the compiler to use
-* `compiler-version`: The version of the compiler to use
-* `spack-packages-version`: Optional, defaults to `main`. The git tag or branch for the `ACCESS-NRI/spack_packages` repository. 
-
-### Usage
-To use, modify the following .yml file and add to your target repository in the `.github/workflows/` directory:
-
-`.github/workflows/workflow.yml`:
-
-```
-name: Example package build testing workflow
-
-on:
-  workflow_dispatch:
-  pull_request:
-  push:
-    branches:
-      - master
-
-jobs:
-  build:
-    uses: access-nri/workflows/.github/workflows/build-package.yml@main
-    with:
-      container-registry: ghcr.io
-      container-name: access-nri/build-${{ github.event.repository.name }}-intel2021.2.0
-      package-name: [your package name]
-      compiler-name: intel
-      compiler-version: 2021.2.0
-      spack-packages-version: main # this doesn't need to be specified, defaults to main
-    permissions:
-      packages: read
-```
-
-See [Reusing workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow) for more info.
+
+### For Model repositories
+
+If you want to use the Model Test Pipeline (and the model is available in `access-nri/spack_packages` and has an associated entry in `containers/models.json`), go to the repo, then the `Actions` tab, then the `New Workflow` button. You should see a section of starter workflows by ACCESS-NRI. Simply add the `Model Build Test Workflow`, and next time there is a PR on that repo, it will test for installability.
+
+### Creation of your own Dependency Images
+
+There is an associated `workflow_dispatch` trigger on `build-and-push-image-build.yml` that allows the creation of your own `base-spack` and `dependency` images. Just make sure that the `spack_packages version` tag exists in the `access-nri/spack_packages` repo.
+
+A [Web UI trigger](https://github.com/ACCESS-NRI/build-ci/actions/workflows/build-and-push-image-build.yml) is available.
+
+## More information
+
+For more of a dev-focussed look at the CI pipeline, see `README-DEV.md`.
```

Comment:
```suggestion
For more of a dev-focussed look at the CI pipeline, see [`README-DEV.md`](https://github.com/ACCESS-NRI/build-ci/blob/main/README-DEV.md).
```
Slightly dodgy, using a link that doesn't exist ... yet. But I'm amongst friends ... right? I wonder if there is a way to do relative links? Hrrrm.